app/facades/foods_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodsFacade
+  class << self
+    def search_by_food(food_input)
+      searched_food_info = FoodsService.search_by_food_db(food_input)
+
+      searched_food_info[:foods].map do |food_info|
+        Food.new(food_info)
+      end
+    end
+
+    def searched_food_count(food_input)
+      FoodsService.search_by_food_db(food_input)[:totalHits]
+    end
```

Comment:
Instead of having two methods here we could do something like this:
```ruby
searched_food_info = FoodsService.search_by_food_db(food_input)
food_data = searched_food_info[:foods].map do |food_info|
        Food.new(food_info)
      end
return [searched_food_info[:totalHits], food_data]
```

This way we return a 2d array with everything we need back to the controller. Then, we can unpack this array like so: 
```ruby
@total_food_count, @foods_info = FoodsFacade.search_by_food(food_input)
```
That's some fancy syntax there!